easybuild/easyconfigs/p/pypmt/pypmt-1.1.0-foss-2022a.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'pypmt'
+version = '1.1.0'
+
+homepage = 'https://git.astron.nl/RD/pmt'
+description = """PMT is a high-level software library capable of 
+    collecting power consumption measurements on various hardware."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+builddependencies = [('CMake', '3.23.1')]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+]
+
+sources = [{
+    'filename': 'pmt-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://git.astron.nl/RD',
+        'repo_name': 'pmt',
+        'tag':  '%(version)s'
+    },
+}]
+
+checksums = ['d6332ce193dfdbc8073ba1fb600d1cebbd0a17c00f650003fab044aa721faf70']
+
+configopts = '-DBUILD_PYTHON_PMT=1'
+
+sanity_check_paths = {
+    'files': ["bin/Rapl-test"],
+    'dirs': [
+        "lib",
+        "lib/python",
+        "include"],
+}
+
```

Comment:
You're not using an easyblock that derives from `PythonPackage`. We privately discussed this, and this is ok, since this installation is quite non-standard for a python package anyway (e.g. doens't end up in the standard python path, with `site-packages` etc). However, it does mean that there is also no default `import pmt` test being done. You should probably specify that as a `sanity_check_commands`. I.e. something like

```
sanity_check_commands = [('python', "-c 'import pmt'")]
```